changelog_5c1ce29804286304a71d9ba4.md

```diff
@@ -2,6 +2,17 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of Apr 29, 2019
+
+- Platform updates:
+  - Chrome 74, ChromeDriver 74
+  - Heroku 7.24.1
+  - Git-lfs 2.7.2
+  - Pip 19.1
+  - Phpunit 7.5.9
+  - Removed Oracle Java 7,9,10
+  - Java 8,11 changed from Oracle Java to openjdk
```

Comment:
Please unify Java changes in a single bullet point, adding a point that there's nothing users need to do.

Reply:
Unified